Data-Science/Kubeflow/MNIST-Digits-Recognition/mnist_katib_tf_kserve_example.ipynb

```diff
@@ -372,10 +376,16 @@
    "outputs": [],
    "source": [
     "# This function converts Katib Experiment HP results to args.\n",
-    "def convert_katib_results(katib_results) -> str:\n",
+    "\n",
+    "from kfp import components\n",
+    "@dsl.component(\n",
+    "    install_kfp_package = False,\n",
+    "    base_image = f\"{os.environ['AIRGAP_REGISTRY']}gcr.io/mapr-252711/kubeflow/python-kfp:3.7-2.7.0\",\n",
```

Comment:
Specifying base image and disabling kfp install are not needed any more - we have merged PR, that allows package installation:

- https://github.hpe.com/hpe/ezaf/pull/2855
- https://github.com/mapr/private-manifests/pull/584
- and most impotent - https://github.com/mapr/private-kubeflow-pipelines/pull/32